src/AWSS3.jl

```diff
@@ -552,7 +558,7 @@ function s3_enable_versioning(aws::AbstractAWSConfig, bucket, status="Enabled";
     return parse(r)
 end
 
-s3_enable_versioning(a; b...) = s3_enable_versioning(global_aws_config(), a; b...)
+s3_enable_versioning(a; kwargs...) = s3_enable_versioning(global_aws_config(), a; kwargs...)
```

Comment:
Looks like this should be updated to `args...`:
```suggestion
s3_enable_versioning(args...; kwargs...) = s3_enable_versioning(global_aws_config(), args...; kwargs...)
```
The `bucket` and `status` function cannot be called without also passing in an `AbstractAWSConfig`:
```julia
julia> methods(s3_enable_versioning)
# 3 methods for generic function "s3_enable_versioning":
[1] s3_enable_versioning(aws::AWS.AbstractAWSConfig, bucket) in AWSS3 at /Users/cvogt/.julia/dev/AWSS3/src/AWSS3.jl:537
[2] s3_enable_versioning(aws::AWS.AbstractAWSConfig, bucket, status; kwargs...) in AWSS3 at /Users/cvogt/.julia/dev/AWSS3/src/AWSS3.jl:537
[3] s3_enable_versioning(a; b...) in AWSS3 at /Users/cvogt/.julia/dev/AWSS3/src/AWSS3.jl:555
```